Calculator/Calculator.cpp

```diff
@@ -161,23 +374,35 @@ double term()
     double left = factorial();
     Token t = ts.get();
     while (true) {
-        switch (t.kind) {
-        case '*':
+        switch (t.kind) 
         {
-            left *= factorial();
-            t = ts.get();
-            break;
-        }
-        case '/':
-        { double d = factorial();
-        if (d == 0) error("divide by zero");
-        left /= d;
-        t = ts.get();
-        break;
-        }
-        default:
-            ts.putback(t); 
-            return left;
+            case '*':
+            {
+                left *= factorial();
+                t = ts.get();
+                break;
+            }
+            case '/':
+            {   
+                double d = factorial();
+                if (d == 0) error("divide by zero");
+                left /= d;
+                t = ts.get();
+                break;
+            }
+            case '%':
```

Comment:
I know what this is, but your summary doesn't mention it

Reply:
yeah, i still have to comment everything and add my new functions to the grammar and instructions etc...